tomlrb.gemspec

```diff
@@ -17,4 +17,6 @@ Gem::Specification.new do |spec|
   spec.files         = %w{LICENSE.txt} + Dir.glob("lib/**/*", File::FNM_DOTMATCH).reject { |f| File.directory?(f) }
   spec.require_paths = ["lib"]
   spec.required_ruby_version = '>= 2.0'
+
+  spec.add_development_dependency "psych", "~> 4"
```

Comment:
Why is this needed now? Sorry I've been away from the ruby world for a while.

Reply:
Ruby 3.1 upgraded Psych from 3 to 4 that includes API change, therefore we upgrade Psych to 4 even when we use Ruby 3.0 to keep code simple.